akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -518,7 +519,7 @@ private[stream] final class GraphInterpreter(
         eventsRemaining -= 1
       }
     } finally {
-      setCurrentInterpreter(previousInterpreter)
+      currentInterpreterHolder(0) = previousInterpreter
```

Comment:
Oh, right, the setter is indeed not used anymore, but I anticipate that we will need both getters in the immediate future—when running interpreters within interpreters for fused stream-of-stream stages.